.github/workflows/torchbench.yml

```diff
@@ -97,4 +97,47 @@ jobs:
       exec_command: bash ./pytorch_blade/benchmark/TorchBench/test_torch_bench.sh cpu full
       push_command: ""
       remote_dev_docker: ""
-    secrets: inherit
\ No newline at end of file
+    secrets: inherit
+
+  TorchBenchAArch64Tiny:
+    if: github.event.schedule != '30 15 * * 5' # daily or dispatch
+    uses: ./.github/workflows/reusable.yml
+    with:
+      name: torch-offcial-benchmark
+      base_image: bladedisc/bladedisc:latest-runtime-torch1.12.0-cpu-aarch64
+      device: cpu_benchmark
```

Comment:
Since not use extra machine for benchmark, we can directly use `aarch64` / `cpu` for aarch64 / cpu benchmark now.